content/recipes/findability/findingCompounds.md

```diff
@@ -0,0 +1,80 @@
+# Finding FAIR Experimental Datasets for Chemical Compounds
+ "Using Wikidata to find experimental data related chemical compounds in a FAIR dataset."
+
+
+ ````{panels_fairplus}
+:identifier_text: FCB053 
+:identifier_link: 'https://w3id.org/faircookbook/FCB053'
+:difficulty_level: 2
+:recipe_type: hands_on
+:reading_time_minutes: 15
+:intended_audience: bioinformatician, data_curator, data_scientist, data_engineer
+:has_executable_code: nope
+:recipe_name: How to find FAIR Datasets
+```` 
+
+## Main Objectives
+
+The goal of this recipe is to find compounds from scientific sources to add to Wikidata. Many different trusted sources and repositories contain research surrounding chemical compounds, and, given the proper licenses, these compounds can be added to Wikidata.
+___
+
+## Dataset Extensions and Identifiers
+
+Here are some datasets types which are simple to add to Wikidata. Each has a well-structured format. 
+
+* SDF file (FairSharing doi:[10.25504/fairsharing.ew26v7](https://doi.org/10.25504/fairsharing.ew26v7))
+
+* csv file (FairSharing [bsg-s001546/](https://fairsharing.org/bsg-s001546/))
+
+## Identifiers
+To add other related chemical identifiers to each compound at least universal identifier must be present. For example: 
+
+* SMILES (FairSharing doi:[10.25504/fairsharing.qv4b3c](https://doi.org/10.25504/fairsharing.qv4b3c))
```

Comment:
The SMILES is not a universal identifier, but can be converted into one with this recipe: https://w3id.org/faircookbook/FCB007